src/Core/SettingsChangesHistory.h

```diff
@@ -88,6 +88,8 @@ static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> sett
     {"24.5", {{"allow_deprecated_functions", true, false, "Allow usage of deprecated functions"},
               {"allow_experimental_join_condition", false, false, "Support join with inequal conditions which involve columns from both left and right table. e.g. t1.y < t2.y."},
               {"output_format_parquet_use_custom_encoder", false, true, "Enable custom Parquet encoder."},
+              {"cross_join_min_rows_to_compress", 0, 10000000, "A new setting."},
+              {"cross_join_min_bytes_to_compress", 0, 1_GiB, "A new setting."},
```

Comment:
Please, improve the description. `A new setting.` is not good, it doesn't tell why these settings were added and what they control